provision.sh

```diff
@@ -177,24 +177,28 @@ if [ ! $SKIP_REQUIREMENTS ] ; then
       exit 1
   else
     # Install virtualenv
-    which -a virtualenv >> /dev/null
+    which -a pipenv >> /dev/null
     if [[ $? != 0 ]] ; then
-      pip install virtualenv
+      pip install pipenv
     fi
+    cd $ROOT/ansible
+    VENV=`pipenv --venv`
+
     # Create a virtualenv for this project and use it for ansible
-    if [ ! -f $ROOT/.virtualenv ]; then
-      virtualenv --python=python2.7 $ROOT/ansible/.virtualenv
+    if [ ! $VENV ]; then
+      pipenv --python=python2.7
```

Comment:
@tharna This requires the host machine to have python 2.7 installed which is a fair assumption I guess. At least it ships with macOS and devs on other platforms can be expected to install it. Maybe add a mention of this to docs?